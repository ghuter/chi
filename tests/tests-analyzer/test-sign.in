Signature :: signature {
	Struct :: struct {
		x: i32;
	}
	Function :: fun ();
}
