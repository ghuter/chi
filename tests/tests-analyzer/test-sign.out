SMODSIGN(Signature, [], (), {SSTRUCT(Struct, x : i32), SSIGN(Function) : void})
